npm.el

```diff
@@ -59,7 +60,8 @@ This will first check to make sure there is a package.json file and then open th
       ("u" "Update"       npm-update)
       ("i" "Install"       npm-install-menu)
       ("r" "Run"       npm-run)
-      ("t" "Test"       jest-popup)]]
+      ("t" "Test"       jest-popup)
+      ("p" "Publish"       npm-publish)]]
```

Comment:
Similar to line 61 where I defined the "install" command, it references the intall _menu_ since there are a bunch of install options. Here you're going to want to reference the `npm-publihs-menu`